compiler/luci/service/src/ShapeInfer_StridedSlice.cpp

```diff
@@ -56,6 +62,35 @@ struct StridedSliceParams
   int16 shrink_axis_mask;
 };
 
+struct StridedSliceContext
+{
+  StridedSliceContext(const luci::CircleStridedSlice *node)
+  {
+    params.begin_mask = node->begin_mask();
+    params.ellipsis_mask = node->ellipsis_mask();
+    params.end_mask = node->end_mask();
+    params.new_axis_mask = node->new_axis_mask();
+    params.shrink_axis_mask = node->shrink_axis_mask();
+
+    input = loco::must_cast<luci::CircleNode *>(node->input());
+    begin = loco::must_cast<luci::CircleConst *>(node->begin());
+    end = loco::must_cast<luci::CircleConst *>(node->end());
+    strides = loco::must_cast<luci::CircleConst *>(node->strides());
+
+    loco::TensorShape input_shape = luci::shape_get(input).as<loco::TensorShape>();
+    input_dims = input_shape.rank();
+  }
+  StridedSliceParams params;
+  luci::CircleNode *input;
+  luci::CircleConst *begin;
+  luci::CircleConst *end;
+  luci::CircleConst *strides;
```

Comment:
Maybe we need initial value for SE score? `= nullptr;`

Reply:
ctor `StridedSliceContext` exist but maybe to make analysis tool happy :)